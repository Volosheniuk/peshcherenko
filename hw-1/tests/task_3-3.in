11110010001010100001000110101111
