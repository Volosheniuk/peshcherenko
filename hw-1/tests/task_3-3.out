F22A11AF
